sdk/search/arm-search/README.md

```diff
@@ -2,7 +2,7 @@
 
 This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure SearchManagement client.
 
-Client that can be used to manage Azure Cognitive Search services and API keys.
+Client that can be used to manage Azure AI Search services and API keys.
```

Comment:
Do we need an ad-hoc release for this update? Or we only need to update the github readme to the latest?

Reply:
My primary interest is in updating the content in our public API reference documentation.  I don't know how often you rebuild docs, but I thought it was weekly.  Maybe that's not the case though. FWIW, the product has been renamed for about 3 weeks now.